src/test/kotlin/de/klg71/keycloakmigration/changeControl/actions/client/AddRoleScopeMappingIntegTest.kt

```diff
@@ -0,0 +1,81 @@
+package de.klg71.keycloakmigration.changeControl.actions.client
+
+import de.klg71.keycloakmigration.AbstractIntegrationTest
+import de.klg71.keycloakmigration.changeControl.actions.MigrationException
+import de.klg71.keycloakmigration.changeControl.actions.role.AddRoleAction
+import de.klg71.keycloakmigration.keycloakapi.model.RoleListItem
+import de.klg71.keycloakmigration.keycloakapi.KeycloakClient
+import de.klg71.keycloakmigration.keycloakapi.clientUUID
+import de.klg71.keycloakmigration.keycloakapi.model.RoleScopeMapping
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.Test
+import org.koin.core.inject
+import java.util.UUID
+
+class AddRoleScopeMappingIntegTest : AbstractIntegrationTest() {
```

Comment:
Can we add a testcase for the undo() method? I know i didn't add it for all migrations but for the new ones I try to be better :D

Reply:
done